internal/shim-sgx/src/handler/mod.rs

```diff
@@ -33,13 +33,13 @@ use core::fmt::Write;
 use core::mem::size_of;
 use core::ptr::read_unaligned;
 use core::ptr::NonNull;
-use libc::{c_int, c_ulong, off_t, size_t};
 
 use sallyport::guest::Handler as _;
 use sallyport::guest::{self, Platform, ThreadLocalStorage};
 use sallyport::item::enarxcall::sgx::{Report, ReportData, TargetInfo, TECH};
 use sallyport::item::enarxcall::SYS_GETATT;
 use sallyport::item::syscall::{ARCH_GET_FS, ARCH_GET_GS, ARCH_SET_FS, ARCH_SET_GS};
+use sallyport::libc::{self, c_int, c_ulong, off_t, size_t};
```

Comment:
Why do we not just use the `core_ffi_c` feature (which we use in sallyport anyway, tracking issue https://github.com/rust-lang/rust/issues/94501) and definitions from `core::ffi` here?

Reply:
We could, it doesn't make a difference, they're just type aliases. I just made this patch as minimally-invasive as possible.